src/Normalizer/Formatter/JsonFormatter.php

```diff
@@ -33,7 +36,11 @@ public function format(mixed $value): void
         } elseif (is_bool($value)) {
             $this->write($value ? 'true' : 'false');
         } elseif (is_scalar($value)) {
-            $this->write(json_encode($value, JSON_THROW_ON_ERROR));
+            /**
+             * @phpstan-ignore-next-line / Due to the new json encoding options feature, it is not possible to let SA
+             *                             tools understand that JSON_THROW_ON_ERROR is always set.
+             */
+            $this->write(json_encode($value, $this->jsonEncodingOptions));
```

Comment:
Should it be this instead (untested)?
```suggestion
            $this->write(json_encode($value, $this->jsonEncodingOptions | JSON_THROW_ON_ERROR));
```

Reply:
I think it's better (performance-wise) to disable the SA check than forcing it at runtime when it has been forced before.